plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -57,6 +58,7 @@ export const EntityPicker = (
   } = props;
   const allowedKinds = uiSchema['ui:options']?.allowedKinds;
   const defaultKind = uiSchema['ui:options']?.defaultKind;
+  const defaultNamespace = uiSchema['ui:options']?.defaultNamespace;
```

Comment:
How did you plan to represent the "false" case? The code here seems to allow undefined, but the code further down does `?? ''` which probably is worse.

Reply:
I admit that I am little bit lost. The ?? '' is the only way how I was able to make it work. If I remove ?? '' which I would expect to be the right way, is incompatible.